src/RA_Core.cpp

```diff
@@ -1379,7 +1379,18 @@ API void CCONV _RA_InvokeDialog(LPARAM nID)
                 sprintf_s(sRichPresenceFile, 1024, "%s%u-Rich.txt", RA_DIR_DATA, g_pCurrentGameData->GetGameID());
 
                 //	Then install it
-                g_RichPresenceInterpreter.ParseRichPresenceFile(sRichPresenceFile);
+                std::string sRichPresence;
+                std::ifstream file(sRichPresenceFile);
```

Comment:
How about a `_ReadBufferFromFile()` function to mirror the use of `_WriteBufferToFile`? That will also make factoring out file operations and other supporting functions out of this large file easier in the future.

Reply:
will do